packages/esm-admin-openconceptlab-app/webpack.config.js

```diff
@@ -1 +1,10 @@
-module.exports = require('openmrs/default-webpack-config');
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
The intention is to use the path aliases to clean up imports in tests. Since we're not using these path aliases in actual app code, we don't need to extend the changes to the webpack configs.